### PR TITLE
Cld 7454 vast api improvements

### PR DESCRIPTION
**Pull request by yaniv-li**

### CODE REVIEW

File: src/video-player.js [hunk 1]
```diff
@@ -287,6 +299,20 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     });
 
 
+    if (Object.keys(options.playerOptions.ads).length > 0 && typeof this.videojs.ima === 'object') {
```
Comment (mikeys) [inline src/video-player.js]: In general looks good, but I assume you'd probably want some way for the user to actually turn this off on demand.

Comment (yaniv-li) [reply src/video-player.js]: if there is no ads option then it turns it off

Comment (mikeys) [reply src/video-player.js]: Once the player has been initialised, is there currently a way to turn it off?

Comment (yaniv-li) [reply src/video-player.js]: Sorry I don't follow

Comment (mikeys) [reply src/video-player.js]: Once you create a player instance with "ads" options to enable ads, is there currently a way to turn it off?

Comment (yaniv-li) [reply src/video-player.js]: no there is no option, it wasn't in the requirements

**Pull request merged**